src/commands/convert.ts

```diff
@@ -78,13 +78,13 @@ export default class Convert extends Command {
       if (document !== undefined) {
         metadata = MetadataFromDocument(document, metadata);
         metadata['from_version'] = document.version();
+        this.specFile = await load(filePath);
```

Comment:
I'm concerned about this extra parsing stuff here. Would it make sense to use the original file (`specFile` var) instead 🤔

Reply:
Nevermind, we can revisit all of that in the general PR.